docs/provider-pkcs11.7.md

```diff
@@ -0,0 +1,262 @@
+% provider-pkcs11(7) | Configuration directives
+
+NAME
+====
+pkcs11-provider - An OpenSSL provider that allows to directly interface
+with pkcs11 drivers.
+
+
+DESCRIPTION
+===========
+
+Starting with version 3.0 the OpenSSL project introduced a new modular
+system to exten OpenSSL that replaces the deprecated Engines modules
+
+Providers[1] are loaded via configuration directives in the openssl
+configuration file (or via direct loading by applications).
```

Comment:
i find the wording here a big cumbersome so I would propose something like this here:
```suggestion
configuration file (or directly loaded by applications).
```